soccer/src/soccer/tutorial/soccer_mom_node.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include <rclcpp/rclcpp.hpp>
+#include <memory>
+#include <string>
+#include <rj_msgs/msg/team_color.hpp>
+#include <std_msgs/msg/string.hpp>
+
+
+using TeamColorMsg = rj_msgs::msg::TeamColor;
+
+class SoccerMom: public rclcpp::Node{
+
+	public:
+    SoccerMom() : rclcpp::Node("soccer_mom") {
+        team_color_subscription_ = this->create_subscription<TeamColorMsg>(
+            "/referee/team_color", rclcpp::QoS(1), std::bind(&SoccerMom::team_color_callback, this, std::placeholders::_1));
+        team_fruit_publisher_ = this->create_publisher<std_msgs::msg::String>("/team_fruit", rclcpp::QoS(1));
+	}
+
+
+	private:
```

Comment:
It is important to adhere to the C++ doc string format whenever you are designing something new. It will be enforced in future projects!